EIPS/eip-inclusionlists.md

```diff
@@ -0,0 +1,170 @@
+# [DRAFT] Inclusion list EIP 
+
+eip: not assigned
```

Comment:
```suggestion
eip: 7547
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

Reply:
done!